packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -73,11 +73,18 @@ class IOSDevices extends PollingDeviceDiscovery {
   }
 }
 
+enum IOSDeviceInterface {
+  none,
```

Comment:
Is `none` exclusively simulators?

Reply:
AFAIK but it's also a fallback in case they add some new type we can't parse.

Reply:
Added a clarifying comment.

Reply:
Thanks